contracts/layer-2/ChampMarketplace.sol

```diff
@@ -88,14 +88,17 @@ contract ChampMarketplace is AccessControlEnumerableUpgradeable {
     // (wallet => Option) mapping of locks to prevent multiple options per wallet
     mapping(address => OptionLock) private _optionLock;
 
-    // Percent fees applied on each sale.
+    // Percent fees applied on each sale. Stores sell and buy fees: 0-128bits buy fee, 128-256bits -> sell fee
     uint256 private _marketplacePercentFees;
```

Comment:
If we split the slot (256bits) into 2 vars of 128 bits, I think it still uses exactly 1 slot and therefore, the new implementation is compatible with smartcontract upgrade

Reply:
I'd even suggest we split this slot into 3 vars to fit the burn fee value as well. Avoids the need to add a new state variable and makes functions like `computeSaleShares` (and thus `acceptSale` also) cheaper to operate since we don't need to pay the (very high) gas cost of loading data from an additional storage slot. Also, a uint256 to store a burn fee value that will most likely always be a single digit number is really overkill.

WDYT?

Reply:
Yeps, a uint8 is enough for each vars. in order to keep the same number of slots, we still need to do something like 

```
uint8 private _buyFee;
uint8 private _sellFee;
uint8 private _burnFee;
uint232 private __EMPTY_SLOT;
```